### PR TITLE
Change flag style command to sub-command style

### PR DESCRIPTION
**Pull request by yujideveloper**
## Overview

I changed command style that from flag style command to sub command style(like the git command).

## Changes
* Introduce a sub command style.
  * e.g. `convergence apply example.schema -c database.yml`
* A flag style command has been deprecated
  * e.g. `convergence -c database.yml -i example.schema --apply`
* Usage of flag style diff command is changed
  * `convergence --diff schema1.rb --dfif schema2.rb` and `convergence --diff=schem1.rb,schem2.rb
` are not work. You need to use `convergence --diff schem1.rb schem2.rb` instead  if you use a flag style command.
   `--apply`, `--dryrun`, and `--export` still work the same way as previously.
  * This is a **breaking change**.
* The dry-run command is changed to a option of the apply command.
* `convergence.rb` no longer loads all modules.

### CODE REVIEW

**Comment by nishio-dens** [comment]
Thank you for your great work!
This PR has breaking changes, and I'll bump up convergence version from 0.2.7 to 1.0.0.

File: lib/convergence/cli.rb [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    Convergence::Command::Export.new(opts, config: config).execute
+  end
+
+  desc 'version', 'print the version'
+  def version
+    require 'convergence/version'
+    puts "version #{Convergence::VERSION}"
+  end
+
+  # TODO: `__fallback` will be removed in a future version(maybe v0.4.0)
```
Comment (yujideveloper) [inline lib/convergence/cli.rb]: Hi, @nishio-dens 

Congratulation on v1.0.0 🎉.

I was thinking next version is v0.3.0 and deprecated codes will be dropped on v0.4.0.
But convergence has been release v1.0.0 now. We should  change this comments.
When do you think drop deprecated code?

Comment (nishio-dens) [reply lib/convergence/cli.rb]: Oh, sorry. I missed this commend.

Let's drop this code when it becomes version 1.1.0.

Comment (yujideveloper) [reply lib/convergence/cli.rb]: I fixes on PR #61

**Pull request merged**